libs/core/src/lib/carousel/carousel.component.scss

```diff
@@ -3,3 +3,16 @@
 fd-carousel:focus {
  outline: 1px dotted;
 }
+
+.fd-carousel__content {
+    > div {
+        touch-action: pan-x;
+        user-select: none;
+        -webkit-user-drag: none;
+        -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+
+        &.fd-carousel__content--horizontal {
+            touch-action: pan-y;
+        }
+    }
+}
```

Comment:
Please move this to fundamental-styles

Reply:
I've created separate bug in fundamental-styles to reflect the carousel component's layout from fundamental-ngx: https://github.com/SAP/fundamental-styles/issues/2574